2021/application.md

```diff
@@ -0,0 +1,68 @@
+# Application process
+Technical writers interested in participating to GSoD with ArviZ should follow these steps:
+
+1. Join the dedicated ArviZ Gitter channel for season of docs: https://gitter.im/arviz-devs/season_of_docs
+1. Send an statement of interest to `arviz.devs@gmail.com` before May 1st. More info about your statement of
+   interest at []().
+1. Fulfill the application requirements before May 10th.
+1. Be available for feedback on your proposal.
+
+## Application requirements
+Only applications from people who perform the couple tasks below will be considered.
+
+The only requirement will be ...
```

Comment:
Do we want to have them submit a simple PR to get started? Send a very short feedback comment on one of the pages in our docs? (with limit of characters/words and everyone doing the same one) Nothing, sending the statement of interest and proposal is enough?

Reply:
I think a PR might not be a good bar as it implies coding and git experience. Perhaps just a summary of writing or a suggestion in at minimum an issue ticket or something that doesnt require git

Reply:
An issue ticket sounds great, it won't only serve as an evaluation of what feedback they can provide, but also to see how clearly can they translate that into actionable input for other people. I think we have enough pages to evaluate, that are short enough to not require too much of their time. We can even ask for feedback on half pages. i.e. read the first half of the label guide and open an issue with your feedback and some actionable suggestions. No need to actually implement the suggestions, someone else may even come around and make them a reality for them

Reply:
Ok so I think then we need

- A proposal very similar to Google Season of Code in that its high level but includes detail on whats proposed, why, a timeline. We should say we'll help review these proposals up until a week before the deadline
- One issue ticket for something we can improve. It can be small but should demonstrate identification of a documentation issue and suggestions how it should be resolved

Reply:
basically yes, but I wouldn't phrase it as similar to summer of code proposal. This is very closely based on official season of docs advise and we should not assume applicants here know about gsoc and using parallelisms with that will help them.

Reply:
Not sure about opening issues is the way to go, maybe keeping the entire proposal private is a better idea. If we want them to "demonstrate identification of a documentation issue and suggestions how it should be resolved" they can write that in the proposal.

Reply:
We can keep everything private, it's no problem